### ISSUE TITLE
Add an optional UI notification to stop ringing after the user issued the 'ring' command

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#311]
```
Hi, as previously discussed in #309 I'd like some sort of button that stops the 
phone from ringing after the ring command has been issued, for example:

Run ring command
Phone begins to ring
Window appears on phone saying something akin to "GTalkSMS is making your phone 
ring" and having a "Stop" button

```

Original issue reported on code.google.com by `azelp...@azelphur.com` on 3 Jan 2013 at 7:41
- Blocked on: #309

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 4 Jan 2013 at 12:38
- Changed title: **Add an optional UI notification to stop ringing after the user issued the 'ring' command**
- Changed state: **Accepted**
- Now blocked on: #309

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 27 Jan 2013 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 125a9f3320e4.
```

Original comment by `yako...@gmail.com` on 27 Jan 2013 at 11:02
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
New version 4.4 Beta 1 is available in the download section
```

Original comment by `Florent....@gmail.com` on 16 Feb 2013 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4
```

Original comment by `Florent....@gmail.com` on 15 Sep 2013 at 3:27
- Changed state: **Deployed**
